src/bento/studyDetailsData.js

```diff
@@ -12,7 +12,7 @@ export const embargoFileIcon = 'https://raw.githubusercontent.com/CBIIT/bento-ic
 export const embargoHeaderIcon = 'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/assets/icons/Icon-embargo-study-header.svg';
 export const headerIcon = 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/master/icdc/images/svgs/Icon-StudiesDetail.svg';
 export const externalIcon = 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/master/icdc/images/svgs/cart_ExternalLink.svg';
-
+export const tab = { items: [{ id: 0, label: 'OVERVIEW', value: 'overview' }, { id: 1, label: 'PUBLICATION', value: 'publication' }] };
```

Comment:
Can you make it custodian configurable ?

Reply:
updated